pipelines/broad/dna_seq/germline/single_sample/wgs/WholeGenomeGermlineSingleSample.changelog.md

```diff
@@ -1,3 +1,14 @@
+# 2.4.5
+2021-06-10
+
+* Updated VerifyBamID to use AppSec base image
+
+# 2.4.4
+2021-06-09
+
+* Updated GoTC base image to AppSec approved 
+* Updated BWA version for GoTC image
```

Comment:
Did we actually change the BWA version? the version looks the same to me (but we will want to consult scientific owners if this does change the version of BWA)

Reply:
It was in the GoTC dockerfile...the release URL that was previously being referenced didn't actually exist. they had some weird version tag....i just rolled it down to 0.7.15 (https://github.com/lh3/bwa/releases/tag/v0.7.15)

probably will be good to double check...could still be the same underlying executable

Reply:
Ok, yeah, that looks fine. I believe that is the same version of [BWA](https://github.com/lh3/bwa/releases)